hyrax/app/views/hyrax/my/_work_action_menu.html.erb

```diff
@@ -0,0 +1,50 @@
+<% ul_id = 'admin-set-action-dropdown-ul-' + document.id %>
+
+<div class="btn-group">
+
+  <button class="btn btn-default btn-sm dropdown-toggle" data-toggle="dropdown" type="button" id="dropdownMenu_<%= document.id %>" aria-haspopup="true" aria-expanded="false" aria-controls="<%= ul_id %>">
+    <span class="sr-only"><%= t("hyrax.dashboard.my.sr.press_to") %> </span>
+    <%= t("hyrax.dashboard.my.action.select") %>
+    <span class="caret" aria-hidden="true"></span>
+  </button>
+
+  <ul role="menu" id="<%= ul_id %>" class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenu_<%= document.id %>">
+
+    <% if can? :edit, document.id %>
+      <li role="menuitem" tabindex="-1">
+        <%= link_to [main_app, :edit, document] do %>
+          <i class="glyphicon glyphicon-pencil" aria-hidden="true"></i>
+          <span> <%= t("hyrax.dashboard.my.action.edit_work") %> </span>
+        <% end %>
+      </li>
+
+      <li role="menuitem" tabindex="-1">
+        <%= link_to [main_app, document],
+                    method: :delete,
+                    data: {
+                      confirm: t("hyrax.dashboard.my.action.work_confirmation", application_name: application_name) } do %>
+          <i class="glyphicon glyphicon-trash" aria-hidden="true"></i>
+          <span> <%= t("hyrax.dashboard.my.action.delete_work") %> </span>
+        <% end %>
+      </li>
+    <% end %>
+
+    <!--
+    # https://github.com/antleaf/nims-mdr-development/issues/402
+    <li role="menuitem" tabindex="-1">
+      <%= display_trophy_link(current_user, document.id) do |text| %>
+        <i class="glyphicon glyphicon-star" aria-hidden="true"></i> <%= text %>
+      <% end %>
+    </li>
+    -->
```

Comment:
Confirmed that the only difference between this file and https://github.com/samvera/hyrax/blob/2.x-stable/app/views/hyrax/my/_work_action_menu.html.erb is commenting out this menuitem.